debian/wb-configs.displace

```diff
@@ -12,7 +12,7 @@
 /etc/hostapd.conf.wb
 /etc/default/hostapd.wb
 /etc/watchdog.conf.wb
-
+/etc/ssh/sshd_config
```

Comment:
why no .wb extension? 
Why no systemd network config here?

Reply:
right, i add ".wb" to sshd_config.
network config is only for stretch and it is included in: https://github.com/contactless/wb-configs/blob/4d9a60cd8eedf5041584387370fd38c94f0c3885/debian/wb-configs-stretch.displace#L1